v2/azure-arm.yaml

```diff
@@ -2300,6 +2344,8 @@ objectModelConfiguration:
         $generatedConfigs:
           ConnectionString: $.Status.ConnectionString
           InstrumentationKey: $.Status.InstrumentationKey
+        $azureGeneratedSecrets:
+          - HockeyAppToken
```

Comment:
Is this a secret or no? If it's on the status then it sortof seems like it can't be a secret (because we're storing it on Status which is unencrypted?)

Reply:
Removed.